content/en/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -14,19 +16,25 @@ the Kubernetes Security Response Committee. See
 [Kubernetes Security and Disclosure Information](/docs/reference/issues-security/security/)
 for more details.
 
-The Kubernetes project publishes a programmatically accessible
-[JSON Feed](/docs/reference/issues-security/official-cve-feed/index.json) of 
-published security issues. You can access it by executing the following command:
-
-{{< comment >}}
-`replace` is used to bypass known issue with rendering ">"
-: https://github.com/gohugoio/hugo/issues/7229 in JSON layouts template
-`layouts/_default/cve-feed.json`
-{{< /comment >}}
+The Kubernetes project publishes a programmatically accessible feed of published
+security issues in [JSON feed](/docs/reference/issues-security/official-cve-feed/index.json)
+and [RSS feed](/docs/reference/issues-security/official-cve-feed/feed.xml)
+formats. You can access it by executing the following commands:
 
+{{< tabs name="CVE feeds" >}}
+{{% tab name="JSON feed" %}}
+[Link to JSON format](/docs/reference/issues-security/official-cve-feed/index.json)
+```shell
+curl -L https://k8s.io/docs/reference/issues-security/official-cve-feed/index.json
+```
+{{% /tab %}}
+{{% tab name="RSS feed" %}}
+[Link to RSS format](/docs/reference/issues-security/official-cve-feed/feed.xml)
 ```shell
-curl -Lv https://k8s.io/docs/reference/issues-security/official-cve-feed/index.json
```

Comment:
Any reason to remove verbose flag?

Reply:
I was mostly wondering why using it? And I thought it was maybe a bit unrelevant, but is there a reason? :)

Reply:
Mainly for "content-type" in header and to verify that redirects (301) happens safely

Reply:
See example for xml file:
```
curl -lv https://localhost:1313/docs/reference/issues-security/official-cve-feed/feed.xml
```

>< Content-Type: application/xml; charset=utf-8

Same thing applies for json file too.

Reply:
I put back the verbose flag if you think it's better this way!